test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -190,7 +196,19 @@ def prepareTestLoader(self, loader):
         old_loadTestsFromNames = loader.loadTestsFromNames
 
         def check_loadTestsFromNames(names, module=None):
-            rv = old_loadTestsFromNames(names, module)
+            tests_dir = names
+            if not self.testfiles:
+                ptl_test_dir = __file__
+                ptl_test_dir = os.path.join(ptl_test_dir.split('ptl')[0],
+                                            "ptl", "tests")
+                user_test_dir = os.environ.get("PTL_TESTS_DIR", None)
+                if user_test_dir and os.path.isdir(user_test_dir):
+                    tests_dir += [user_test_dir]
+                if os.path.isdir(ptl_test_dir):
+                    tests_dir += [ptl_test_dir]
+                rv = old_loadTestsFromNames(tests_dir, module)
+            else:
+                rv = old_loadTestsFromNames(names, module)
```

Comment:
you set tests_dir to name before, so you can just call "rv = old_loadTestsFromNames(tests_dir, module)" right?

Reply:
Agree.